python/ray/serve/config.py

```diff
@@ -58,6 +60,9 @@ class BackendConfig(BaseModel):
     max_concurrent_queries: Optional[int] = None
     user_config: Any = None
 
+    experimental_graceful_shutdown_wait_loop_s: PositiveFloat = 2.0
+    experimental_graceful_shutdown_timeout_s: PositiveFloat = 20.0
+
```

Comment:
Made shutdown parameters per backend instead of the global constants. (in k8s, `terminationGracePeriodSeconds` is per pod config anyway)